apps/server/package.json

```diff
@@ -7,10 +7,10 @@
   "dependencies": {
     "@googleapis/sheets": "^5.0.5",
     "cors": "^2.8.5",
-    "dotenv": "^16.0.1",
-    "express": "^4.21.1",
-    "express-static-gzip": "^2.2.0",
-    "express-validator": "^7.2.0",
+    "dotenv": "^16.3.1",
+    "express": "^4.18.2",
+    "express-static-gzip": "^2.1.7",
+    "express-validator": "^7.0.1",
```

Comment:
question: is there a reason for us to be downgrading server dependencies. This is not something we want to do

Reply:
Reverted in https://github.com/cpvalente/ontime/pull/1438/commits/7630b4d5e7eadf88bafd4ec88b23079375fd2f0d